src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -199,10 +201,6 @@ public function has(string $id): bool
     /**
      * Gets a service.
      *
-     * @throws ServiceCircularReferenceException When a circular reference is detected
-     * @throws ServiceNotFoundException          When the service is not defined
-     * @throws \Exception                        if an exception has been thrown when the service has been resolved
```

Comment:
I would simply remove this line and nothing else.

Reply:
`@throws` is to document exceptions that can be thrown under the responbility of the called method. Re-thrown exceptions don't fit this definition so this `@throws` is just wrong. Wrapping them all won't provide anything more.

Reply:
To me the purpose of the `@throws` tag is to know which Exceptions I should catch when doing
```
$container->get()
```
That's also how static analysis like PHPStan works.

But I'm ok removing the `\Exception` line, it solves my issue.
Pr is update